miio/integrations/humidifier/zhimi/airhumidifier.py

```diff
@@ -108,23 +110,46 @@ def temperature(self) -> Optional[float]:
         return None
 
     @property
+    @sensor("Humidity", unit="%", device_class="humidity")
     def humidity(self) -> int:
         """Current humidity."""
         return self.data["humidity"]
 
     @property
+    @switch(
+        name="Buzzer",
+        icon="mdi:volume-high",
+        setter_name="set_buzzer",
+        device_class="switch",
+        entity_category="config",
+    )
     def buzzer(self) -> bool:
         """True if buzzer is turned on."""
         return self.data["buzzer"] == "on"
 
     @property
+    @setting(
+        name="Led Brightness",
+        icon="mdi:brightness-6",
+        unit="something",
```

Comment:
Instead of doing this, please change the setting definition to default to `None` (or whatever is the expected unit for these per default).